### PR TITLE
Rename kaleido-io/ethconnect to hyperledger-labs/firefly-ethconnect

### PR DESCRIPTION
**Pull request by jimthematrix**

### CODE REVIEW

**Review by nguyer — APPROVED** [review]

**Comment by nguyer** [comment]
@vdamle It looks like there may be an import that was missed. The build is currently failing with:
```
package github.com/hyperledger-labs/firefly-ethconnect
	imports github.com/hyperledger-labs/firefly-ethconnect/cmd
	imports github.com/hyperledger-labs/firefly-ethconnect/internal/rest
	internal/rest/leveldbreceipts.go:26:2: use of internal package github.com/kaleido-io/ethconnect/internal/errors not allowed
package github.com/hyperledger-labs/firefly-ethconnect
	imports github.com/hyperledger-labs/firefly-ethconnect/cmd
	imports github.com/hyperledger-labs/firefly-ethconnect/internal/rest
	internal/rest/leveldbreceipts.go:27:2: use of internal package github.com/kaleido-io/ethconnect/internal/kvstore not allowed
```

**Review by nguyer — CHANGES_REQUESTED** [review]
@vdamle It looks like there may be an import that was missed. The build is currently failing with:
```
package github.com/hyperledger-labs/firefly-ethconnect
	imports github.com/hyperledger-labs/firefly-ethconnect/cmd
	imports github.com/hyperledger-labs/firefly-ethconnect/internal/rest
	internal/rest/leveldbreceipts.go:26:2: use of internal package github.com/kaleido-io/ethconnect/internal/errors not allowed
package github.com/hyperledger-labs/firefly-ethconnect
	imports github.com/hyperledger-labs/firefly-ethconnect/cmd
	imports github.com/hyperledger-labs/firefly-ethconnect/internal/rest
	internal/rest/leveldbreceipts.go:27:2: use of internal package github.com/kaleido-io/ethconnect/internal/kvstore not allowed
```

File: internal/rest/leveldbreceipt_test.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/kaleido-io/ethconnect/internal/kvstore"
+	"github.com/hyperledger-firefly/firefly-ethconnect/internal/kvstore"
```
Comment (nguyer) [inline internal/rest/leveldbreceipt_test.go:27]: I think these should be `hyperledger-labs` not `hyperledger-firefly`

**Review by nguyer — CHANGES_REQUESTED** [review]

File: internal/rest/leveldbreceipt_test.go (hunk 1 above)
Comment (vdamle) [reply internal/rest/leveldbreceipt_test.go:27]: 🤦🏼

**Review by nguyer — APPROVED** [review]

**Pull request merged**